ParkingLot/ParkBoy.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace ParkingLot
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public class ParkBoy
+    {
+        public ParkBoy()
+        {
+            this.ParkLots.Add(new ParkLot(10));
+        }
+
+        public List<Ticket> Tickets { get; set; } = new List<Ticket>();
+
+        public List<Customer> Customers { get; set; } = new List<Customer>();
+
+        public List<ParkLot> ParkLots { get; set; } = new List<ParkLot>();
+
+        public int CurrentParkLot { get; set; }
+
+        public virtual List<Ticket> ParkCar(List<Customer> customers)
+        {
+            int parkLotIndex = 0;
+            int usedPosition = 0;
+            foreach (var customer in customers)
+            {
+                if (parkLotIndex >= ParkLots.Count)
+                {
+                    throw new ParkingLotException("Not enough position.");
+                }
+
+                this.CurrentParkLot = parkLotIndex;
+
+                if (Tickets.Any(t => t.Id == customer.CarID) || customer == null)
+                {
+                    continue;
+                }
+
+                // my system just gererate the ticketID === carID
+                Tickets.Add(new Ticket(customer.CarID, true));
+                customer.TicketID = customer.CarID;
+                customer.HasTicket = true;
+
+                usedPosition++;
+                if (usedPosition >= ParkLots[parkLotIndex].Capacity)
+                {
+                    parkLotIndex++;
+                    usedPosition = 0;
+                }
+            }
+
+            this.Customers = customers;
+            return Tickets;
+        }
```

Comment:
break up some functions

Reply:
Especially, the number of car stored in parkingLots should put into the class of ParkingLot